aws/table_aws_transfer_server.go

```diff
@@ -0,0 +1,260 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/transfer"
+	"github.com/aws/aws-sdk-go-v2/service/transfer/types"
+
+	transferv1 "github.com/aws/aws-sdk-go/service/transfer"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsTransferServer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_transfer_server",
+		Description: "AWS Transfer Server",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("server_id"),
+			Hydrate:    getTransferServer,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listTransferServers,
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(transferv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) for the server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "server_id",
+				Description: "The system-assigned unique identifier for the server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "domain",
+				Description: "Specifies the domain of the storage system that is used for file transfers.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "identity_provider_type",
+				Description: "The mode of authentication for a server.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "endpoint_type",
+				Description: "Specifies the type of VPC endpoint that your server is connected to.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "logging_role",
+				Description: "Specifies the type of VPC endpoint that your server is connected to.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "state",
+				Description: "The condition of the server that was described.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "user_count",
+				Description: "Specifies the number of users that are assigned to a server.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "certificate",
+				Description: "Specifies the ARN of the Amazon Web ServicesCertificate Manager (ACM) certificate.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getTransferServer,
+				Transform:   transform.FromField("Certificate"),
```

Comment:
@jramosf, do we require the transform function here? I believe the default [transform function](https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/plugin.go#L20) will handle that conversion for us (from snake case to Upper Camel Case).

Reply:
Please apply the same update to other columns wherever it is applicable.